docker-entrypoint.sh

```diff
@@ -2,10 +2,14 @@
 
 [ -z "$LISTEN_PORT" ] && export LISTEN_PORT=8080
 
+[ -z "$CACHE_MAX_SIZE" ] && export CACHE_MAX_SIZE=100m
+
+[ -z "$CACHE_TTL" ] && export CACHE_TTL=24h
```

Comment:
Please use a smaller default, we update often qontract-server

Reply:
Do you mean the bundles we load from S3?

Reply:
yes, I mean the bundles we load to S3 in the watched bucket.

Reply:
the idea is to set a very long TTL, let nginx lru delete unused cache items, as our `graphqlsha/{sha}` response is immutable based on payload.

Reply:
Ah, that's a different approach I was understanding. Objects in the cache will be deleted even if they're still valid since they're unlikely to be used.

@bhushanthakur93 Please document this strategy somewhere then.

Reply:
Added a section in README